Bump up winit to 0.24

dmyyy opened the pull request:
Fixes: https://github.com/olson-sean-k/plexus/issues/114

olson-sean-k reviewed (commented):
Thanks for the PR! I have just one small nitpick regarding formatting. Other than that, I just fixed numerous CI failures on `master` at ea7340f and this should probably be rebased.

dmyyy commented:
> Thanks for the PR! I have just one small nitpick regarding formatting. Other than that, I just fixed numerous CI failures on `master` at [ea7340f](https://github.com/olson-sean-k/plexus/commit/ea7340fe2d89db05c982441237c8bdc1c95a4c39) and this should probably be rebased.

Changed due to cargo auto format - should be fixed now. I'm not sure I understand why this needs to be rebased assuming you merged everything into master - it's saying there's no conflicts with the base branch.

olson-sean-k commented:
> I'm not sure I understand why this needs to be rebased...

Ah, my bad. I thought CI would run directly against the PR commit without picking up the more recent fixes from `master`, but it looks like the workflow runs against the merge. No rebase necessary. :smile:
